.github/workflows/dockerimage.yml

```diff
@@ -23,9 +23,15 @@ jobs:
     steps:
     - name: Checkout che-dashboard-next source code
       uses: actions/checkout@v2
-    - name: Build the Docker image
-      run: docker build . --file apache.Dockerfile --tag ${IMAGE_FULL}
+    - name: Docker Buildx
+      uses: crazy-max/ghaction-docker-buildx@v3
+      with:
+        buildx-version: latest
+    - name: Docker prepare
+      run: docker image prune -a -f
     - name: Docker Login
       run: docker login -u "${{ secrets.QUAY_USERNAME }}" -p "${{ secrets.QUAY_PASSWORD }}" quay.io
-    - name: Push the Docker image
-      run: docker push ${IMAGE_FULL}
+    - name: Build and push the Docker image(s)
+      run: docker buildx build --platform linux/amd64,linux/s390x --cache-from="type=registry,ref=${CACHE_IMAGE_FULL}" --cache-to="type=registry,ref=${CACHE_IMAGE_FULL},mode=max" -t ${IMAGE_FULL}  -f apache.Dockerfile --push .
```

Comment:
@shahidhs-ibm  It is not clear for me. 
I think we should add the **CACHE_IMAGE_FULL** into **env**.
Something like this:

```
    env:
      IMAGE_FULL: quay.io/che-incubator/che-dashboard-next:next
      CACHE_IMAGE_FULL: docker.io/cheincubator/che-dashboard-next:cache
```
Am I right?

Reply:
@olexii4 Thank you for the review. Nice catch!! :)
So we have option to add CACHE_IMAGE_FULL into env or to remove them completely and use the default values. I am OK with both. Your suggestion?

Reply:
I will prefer to add the CACHE_IMAGE_FULL as in the example.

Reply:
@olexii4 Incorporated the code review comments. Please check.